pkg/rpmdb.go

```diff
@@ -1,10 +1,11 @@
 package rpmdb
 
 import (
-	"github.com/knqyf263/go-rpmdb/pkg/bdb"
-	dbi "github.com/knqyf263/go-rpmdb/pkg/db"
-	"github.com/knqyf263/go-rpmdb/pkg/ndb"
-	"github.com/knqyf263/go-rpmdb/pkg/sqlite3"
+	"context"
```

Comment:
standard library imports should be before regular ones, with a line separation. See other files in repo, they all abide by this conversion